.buildkite/hooks/pre-exit

```diff
@@ -7,5 +7,6 @@ echo "Attempting to clean up docker containers for buildkitejob=$BUILDKITE_JOB_I
 matching_containers=$(docker ps --filter "label=buildkitejob=$BUILDKITE_JOB_ID" --format "{{.ID}}")
 if [ -n "$matching_containers" ]; then
     echo "Killing docker containers $matching_containers"
+    # shellcheck disable=SC2086
```

Comment:
I think we want word-splitting here. To avoid needing it, we'd have to refactor this a bit.